### PR TITLE
Add pySOT recipe

### PR DESCRIPTION
**Pull request by tadeu**

### CODE REVIEW

**Comment by tadeu** [comment]
@conda-forge/staged-recipes, I think this is ready to be reviewed/merged ;)

File: recipes/pysot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{% set name = "pySOT" %}
+{% set version = "0.1.36" %}
+{% set sha256 = "043655621466d89328cac80a5c87f050c4b4708a79ec29b5912b79c76393504a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/pysot/meta.yaml]: You can drop this line.

File: recipes/pysot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (ocefpaf) [inline recipes/pysot/meta.yaml]: We are moving towards `script: python -m pip install --no-deps --ignore-installed .` (docs will be updated soon)

File: recipes/pysot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (ocefpaf) [inline recipes/pysot/meta.yaml]: If you move to the new script line this must be changed to `pip`.

File: recipes/pysot/meta.yaml (hunk 2 above)
Comment (nicoddemus) [reply recipes/pysot/meta.yaml]: Nice to know, makes perfect sense. 👍

**Pull request merged**